components/cam/src/dynamics/se/stepon.F90

```diff
@@ -598,6 +600,7 @@ subroutine stepon_final(dyn_in, dyn_out)
    ! Deallocate variables needed for the FV physics grid
    if (fv_nphys > 0) then
       call fv_physgrid_final()
+      if (se_fv_phys_remap_alg == 1) call gfr_finish()
```

Comment:
Similar to the comment above, would it make sense to move this into fv_physgrid_final()?

Reply:
Thanks; I'll do both.